src/server/debug/server/debugstar/debugstar.go

```diff
@@ -19,8 +20,16 @@ import (
 
 	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
 	"github.com/pachyderm/pachyderm/v2/src/internal/log"
+	"github.com/pachyderm/pachyderm/v2/src/internal/promutil"
 	ourstar "github.com/pachyderm/pachyderm/v2/src/internal/starlark"
+	"github.com/pachyderm/pachyderm/v2/src/internal/starlark/lib/k8s"
 	"go.starlark.net/starlark"
+	"k8s.io/client-go/dynamic"
+	dfake "k8s.io/client-go/dynamic/fake"
+	"k8s.io/client-go/kubernetes"
+	kfake "k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/kubectl/pkg/scheme"
```

Comment:
Nit: I like to arrange imports so that stdlib packages are in one section, then third-party packages, then our public packages, then our internal packages. This has absolutely no impact on correctness or anything, I just think it reads well.

Reply:
There is no tool that enforces this; goimports will keep doing it if it's already done, but there is no way to mandate that for new packages.  So I think it will be annoying if we make it a rule and there is no way to check if you did it right or not.

Reply:
I did make this change however ;)